sds_data_manager/lambda_code/SDSCode/dependency_config.csv

```diff
@@ -59,8 +59,7 @@ hi, l1b, 45sensor-de, hi, l1c, 45sensor-pset, HARD, DOWNSTREAM
 
 # <---- HIT Dependencies ---->
 
-hit, l0, sci, hit, l1a, sci, HARD, DOWNSTREAM
-hit, l0, hk, hit, l1a, hk, HARD, DOWNSTREAM
+hit, l0, raw, hit, l1a, all, HARD, DOWNSTREAM
```

Comment:
With this change, will cli be able to create the hk product without the science product while that code is still under development?

Reply:
The HIT L0 with a `raw` descriptor will now trigger the L1A processing like the `hk` and `sci` files did. So it can have one or both inside the L0. Since the HIT sample data is the same date (I think), but are split up by `hk` and `sci`, you'll just need to keep track on your end which is which and you won't be able to have both in the SDC at the same time